src/chapters/enums/enums-in-models.rst

```diff
@@ -0,0 +1,191 @@
+Enums in Model Classes
+======================
+
+One application of enum types is to represent categories of objects. We will take this approach in our ``coding-events`` application to categorize events based on their type, such as *conference* or *meetup*.
```

Comment:
I believe we're calling the C# version of the app `CodingEvents`, not `coding-events`, correct?

Reply:
Fixed!